File: exam-portal-frontend/src/app/Question.ts
export interface Question{
  questId: number,
  content: string,
  image: string,
  option1: string,
  option2: string,
  option3: string,
  option4: string,
  answer: string,
  quiz: {
    title: string,
    description: string,
    maxMarks: number,
    numberOfQuestions: number,
    active: boolean,
    category: {
      cid: number,
      title : string,
      description : string,
    },
    qid: number
  }
}

File: exam-portal-backend/src/main/java/com/regulus/examportalbackend/models/Question.java
package com.regulus.examportalbackend.models;

import lombok.*;

import javax.persistence.*;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@Entity
@Table(name = "question")
public class Question {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long questId;

    @Column(length = 5000)
    private String content;

    private String image;

    private String option1;

    private String option2;

    private String option3;

    private String option4;

    private String answer;

    @ManyToOne(fetch = FetchType.EAGER)
    private Quiz quiz;
}
